doxygen-awesome.css

```diff
@@ -1791,7 +1794,7 @@ table.memberdecls .memItemRight  {
 }
 
 table.memberdecls .mdescLeft, table.memberdecls .mdescRight {
-    background: none;
+    background: var(--fragment-desc-background);
```

Comment:
I see a problem with this. It doesn't work well with the current design of the member overview. The block above has rounded corners and creates a small gap, which doesn't look nice IMO:

<img width="497" src="https://user-images.githubusercontent.com/21294002/210264047-0cb5910d-2906-490e-bc45-1916cef89013.png">


I'd like the background coloring of the brief description to be an option that can be enabled, but I'm not sure how this could be implemented atm. :/

Reply:
could be solved by removing `border-right/left-radius` for both `.memItem` left & right respectively, 
also i'm not sure about border color (bottom) should it be left as it is, or removed too and added to mdesc's bottom as shown here:
![image](https://user-images.githubusercontent.com/90788311/210469085-f35b57de-6ebd-4563-ac51-038a0d4f1ba1.png)

or should it be just like this, no borders at all:
![image](https://user-images.githubusercontent.com/90788311/210469481-80db088f-d827-4a23-a59c-07d7985c1c94.png)

what do you think? ( and i truly appreciate the detailed feedback, thanks :) !)